src/tests/test_dynamic_api/test_temporal_actions.py

```diff
@@ -222,6 +223,26 @@ def test_correct_handling_of_extra_url_params(
         }
 
 
+@pytest.mark.django_db
+def test_temporal_auth(api_client, fetch_auth_token, temporal_auth_model, filled_router):
+    """Test the interaction between temporal views and authentication.
+
+    In temporalauth/things, "valid" refers to "start" and "stop".
+    The latter has an "auth" that requires a scope.
+    """
+
+    url = reverse("dynamic_api:temporalauth-things-list")
+
+    response = api_client.get(url, {"valid": "2014-05-01"})
+    assert response.status_code == HTTP_403_FORBIDDEN
```

Comment:
wellicht aardig om te checken wat de body is.. aangezien we dat laatst bij een andere tests ook zagen stukgaan..

Reply:
Punt is, er is niet zo veel te checken, want de dataset waarop getest wordt is leeg...